lib/src/widgets/search_bar.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+
+import '../constants/strings.dart';
+import '../constants/text_style.dart';
+import '../models/coins.dart';
+import '../screens/detail_screen/detail_screen.dart';
+
+class SearchBar extends StatefulWidget {
+  List<Coins> datalist;
+
+  SearchBar(this.datalist);
+
+  @override
+  _SearchBarState createState() => _SearchBarState();
+}
+
+class _SearchBarState extends State<SearchBar> {
+  String _displayStringForOption(Coins option) => option.name!;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: EdgeInsets.all(20),
+      decoration: BoxDecoration(
+          color: Colors.white,
+          borderRadius: BorderRadius.all(Radius.circular(20))),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceAround,
+        children: [
+          SizedBox(
+            width: 20,
+          ),
+          Icon(
+            Icons.search,
+            size: 30,
+            color: Colors.blueAccent,
+          ),
+          SizedBox(
+            width: 10,
```

Comment:
same as above
and sizedbox may in a single line

Reply:
fixed